persistent/tests/test_timestamp.py

```diff
@@ -47,10 +48,10 @@ def test_fromutc(self):
         self.assertTrue(utc.fromutc(source) is source)
 
 
-class pyTimeStampTests(unittest.TestCase):
+class TimeStampTests(unittest.TestCase):
 
     def _getTargetClass(self):
-        from persistent.timestamp import pyTimeStamp
+        from persistent.timestamp import TimeStampPy as pyTimeStamp
         return pyTimeStamp
```

Comment:
I'd change the next line to `return TimeStampPy` instead of using `import ... as` here.